packages/bolt-interactions/README.md

```diff
@@ -175,6 +175,46 @@ interactionFlow<FlowState>((flow, app) => {
 
 Create a [flow listener](#flow-listener) that listens for [actions](https://slack.dev/bolt/concepts#action-listening)
 
+### flow.view
+
+In handling views, you'll need to pass an interactionId as the callback and reference the same string in a `flow.view` action handler. An example of handling view submissions with a view that is opened when an overflow action is clicked:
+
+```typescript
+interactionFlow<FlowState>((flow, app) => {
+  const callback_id = 'edit';
+
+  flow.action<BlockOverflowAction>(
+    'openEdit',
+    async ({
+      action: { action_id },
+      body: { trigger_id },
+      context: { token, interactionIds },
+    }) => {
+      await flow.client.views.open({
+        trigger_id,
+        token,
+        view: {
+          type: 'modal',
```

Comment:
For later, thinking out loud, should "modal" be a higher level block? Laik, should we have "blocks" for surfaces? Maybe? Probably.